code/modules/clothing/suits/ego_gear/he.dm

```diff
@@ -343,3 +343,49 @@
 	attribute_requirements = list(
 							JUSTICE_ATTRIBUTE = 40
 							)
+
+/obj/item/clothing/suit/armor/ego_gear/swindle
+	name = "swindle"
+	desc = "All-natural snake oil! Cleans the skin, removes pimples, impetigo, and other defects!"
+	icon_state = "swindle"
+	armor = list(RED_DAMAGE = 70, WHITE_DAMAGE = -30, BLACK_DAMAGE = -30, PALE_DAMAGE = 0) // 10
```

Comment:
If this wasn't glued into the wishing well, I'd have issues with this, but honestly I think this is fine. Same with the one below.